### DISCUSSION TITLE
Svelte and Nextjs

### DISCUSSION THREAD

**Discussion by mmailaender** [steven-tey/novel#33] [General]
Novel is atm an implementation for Nextjs. Vercel is supporting Nextjs and Svelte. Have you thought about supporting mid-term/long-term also Svelte? :)

**Comment by steven-tey** [comment]
It looks like Tiptap does work with Svelte: https://tiptap.dev/installation/svelte

So yes, this is definitely possible! It's not a top priority right now but contributions welcome! 😄

**Comment by mmailaender** [comment]
Ok, great - Do you have some ideas/preferences about sharing some of the code basis between Nextjs and Svelte? I think I would be not a big fan of simply rewriting the whole code in svelte from the perspective of maintenance.
